src/main/java/org/cbio/gdcpipeline/reader/CnaReader.java

```diff
@@ -105,6 +124,9 @@ private void readFile(File cnaFile, ExecutionContext executionContext) throws Ex
             if (cytobandIndex != -1) {
                 fields.remove(cytobandIndex);
             }
+            for (Integer indexToSkip : indicesToSkip) {
+                fields.remove(indexToSkip);
```

Comment:
same comment here as above. Maybe have a different list of values for the current line that will hold the values to keep and write to the file instead of removing them like this.